py/server/deephaven/execution_context.py

```diff
@@ -36,6 +37,10 @@ class ExecutionContext(JObjectWrapper, ContextDecorator):
     def j_object(self) -> jpy.JType:
         return self.j_exec_ctx
 
+    @property
+    def update_graph(self) -> _JUpdateGraph:
```

Comment:
Two problems:
1) The API is exposing Java objects to the user.
2) The type hint is referring to a private variable.